src/utils/carInputValidation.js

```diff
@@ -0,0 +1,26 @@
+import { RULES } from "../constants/messages.js";
+import { ERROR_MESSAGES } from "../constants/messages.js";
```

Comment:
```suggestion
import { RULES, ERROR_MESSAGES } from "../constants/messages.js";
```
같은 경로에서 `import`하는 것 같아서 하나로 생략 가능할 것 같아요~!